test/Client.test.ts

```diff
@@ -94,7 +94,8 @@ describe('Client', () => {
         const aliceBob = await alice.streamConversationMessages(bob.address)
 
         // alice sends intro
-        await alice.sendMessage(bob.address, 'hi bob!')
+        const sentMessage = await alice.sendMessage(bob.address, 'hi bob!')
+        assert.equal(sentMessage.content, 'hi bob!')
```

Comment:
should we check that the Id was filled in, or whatever was the thing the devs were looking for?

Reply:
Sure. I added some more assertions just to make sure